README.md

```diff
@@ -60,3 +60,32 @@ We collect messages to provide platform without abuse, advertisement.
 - information that break copyright law (or any of it analogue) ©
 - advertisement
 - and staff that can violate law
+
+## Scenario of usage
+
+- `/start` - user get a random identity and get fake identity, status of searching is false
+- `/find_companion` - user status of searching set to true. If match will be found user will get message with companion identity
```

Comment:
Do I need to confirm incoming chat inquiry?

Reply:
After we found the chat, the conversation starts automatically, but the user can stop chatting at any moment of time by typing the command `/stop`